org.freedesktop.Sdk.Extension.swift.json

```diff
@@ -0,0 +1,206 @@
+{
+    "id": "org.freedesktop.Sdk.Extension.swift",
+    "branch": "23.08",
+    "runtime": "org.freedesktop.Sdk",
+    "build-extension": true,
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.llvm16"
+    ],
+    "runtime-version": "23.08",
+    "separate-locales": false,
+    "appstream-compose": false,
+    "build-options": {
+        "prefix": "/usr/lib/sdk/swift",
+        "prepend-pkg-config-path": "/usr/lib/sdk/swift/lib/pkgconfig/",
+        "prepend-path": "/usr/lib/sdk/swift/bin/:/usr/lib/sdk/llvm14/bin",
+        "prepend-ld-library-path": "/usr/lib/sdk/llvm14/lib:/usr/lib/sdk/swift/lib"
+    },
+    "cleanup": [
+        "/share/info",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "libedit",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.thrysoee.dk/editline/libedit-20221030-3.1.tar.gz",
+                    "sha256": "f0925a5adf4b1bf116ee19766b7daa766917aec198747943b1c4edf67a4be2bb"
+                }
+            ],
+            "cleanup": [
+                "/include",
+                "/lib/pkgconfig",
+                "/share/man"
+            ]
+        },
+        {
+            "name": "swift",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.swift.org/swift-5.9.1-release/centos7/swift-5.9.1-RELEASE/swift-5.9.1-RELEASE-centos7.tar.gz",
+                    "sha256": "85f76d092e50794cafc45229d6534c02abb25f4f62f8d923c83b2e7c08a6eada"
+                }
+            ],
+            "build-commands": [
+                "mkdir -p /usr/lib/sdk/swift",
+                "cp -R usr/* /usr/lib/sdk/swift",
+                "sed -i s/libedit.so.2/libedit.so.0/g /usr/lib/sdk/swift/bin/lldb-server",
```

Comment:
this is really really dirty to use sed to change the soversion because they don't match.